attributes/default.rb

```diff
@@ -69,8 +69,12 @@
 default['hops']['protobuf_url']                = "https://protobuf.googlecode.com/files/protobuf-2.5.0.tar.gz"
 default['hops']['hadoop_src_url']              = "https://archive.apache.org/dist/hadoop/core/hadoop-" + node['hops']['version'] + "/hadoop-" + node['hops']['version'] + "-src.tar.gz"
 default['hops']['nn']['http_port']             = 50070
+default['hops']['nn']['https_port']            = 50470
```

Comment:
We already have some attributes for binding ports, check https://github.com/logicalclocks/hops-hadoop-chef/blob/1.3/attributes/default.rb#L317